src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String SEPARATOR = ",";
+    private static final int INDEX_NAME = 0;
+    private static final int INDEX_AMOUNT = 1;
```

Comment:
Better to use naming according to general reading habits. INDEX_NAME and INDEX_AMOUNT looks more like "name of index" and "amount of index" as INDEX is first so it's main.
NAME_INDEX and AMOUNT_INDEX are more understandable